connectors/airwatch/src/main/java/com/vmware/connectors/airwatch/AirWatchController.java

```diff
@@ -272,7 +273,9 @@ public AirWatchController(WebClient rest, CardTextAccessor cardTextAccessor,
                 .uri(gbSession.getBaseUrl() + "/catalog-portal/services/api/entitlements?q={appName}", appName)
                 .cookie("USER_CATALOG_CONTEXT", gbSession.getEucToken())
                 .retrieve()
-                .bodyToMono(JsonDocument.class)
+                .bodyToMono(String.class)
+                .map(s -> Configuration.defaultConfiguration().jsonProvider().parse(s))
```

Comment:
This is not ideal, however, it corrects the problems preventing a demo on dev right now.

I'm not sure which, but 1 of the 3 happened:
* the greenbox actions weren't tested (I'm pretty sure they were, b/c we have a video of this working)
* the greenbox catalog changed their content-type from application/json to application/hal+json (breaking these calls)
* the change to spring webclient broke this (this one seems really plausible b/c the [JsonDocumentHttpMessageConverter](https://github.com/vmware/connectors-workspace-one/blob/9fdd4cc1ae4102198eeb74a2c461d3b914f8302e/common/core/src/main/java/com/vmware/connectors/common/json/JsonDocumentHttpMessageConverter.java#L30) seems like it would handle hal+json)

Reply:
JsonDocumentHttpMessageConverter isn't used in the webclient code.  It is legacy and only used by the APF project at the moment.  [JsonDocumentDecoder](https://github.com/vmware/connectors-workspace-one/blob/9fdd4cc1ae4102198eeb74a2c461d3b914f8302e/common/core/src/main/java/com/vmware/connectors/common/json/JsonDocumentDecoder.java#L39) is what is currently handling this and needs to be corrected.